bench/workgen/workgen.h

```diff
@@ -494,7 +494,9 @@ struct Workload {
 	os << "]";
     }
     void create_table(const std::string& uri);
+    void drop_table(const std::string& uri);
```

Comment:
Since workgen is not using WT create or drop operations, maybe it is better to call these something else, maybe add_table, remove_table?